{"name": "test_002", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test001.Test_Abc object at 0x000001BE97E98948>\n\n    @allure.severity(allure.severity_level.BLOCKER)\n    def test_002(self):\n        allure.attach('我是测试002的描述～～～','描述')\n>       assert False\nE       assert False\n\ntest_script\\test001.py:19: AssertionError"}, "attachments": [{"name": "描述", "source": "55178c0d-dbc3-43d7-809b-a6c83c06275d-attachment.attach"}], "start": 1565580209547, "stop": 1565580209553, "uuid": "49c7c914-093f-4c11-b4dd-23f3de7262dd", "historyId": "a41dbd6a9e748140886031f46d2eb4ec", "fullName": "test_script.test001.Test_Abc#test_002", "labels": [{"name": "severity", "value": "blocker"}, {"name": "parentSuite", "value": "test_script"}, {"name": "suite", "value": "test001"}, {"name": "subSuite", "value": "Test_Abc"}, {"name": "host", "value": "DESKTOP-H1SKJGO"}, {"name": "thread", "value": "13416-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_script.test001"}]}